staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -53,7 +54,7 @@ import (
 
 var (
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
-	registry             = registered.NewOrDie("")
+	Registry             = registered.NewOrDie("")
```

Comment:
Still reading, but I might have tried to go the other way.  I haven't seen value from the registry.  I think I'd enable anything and everything in a scheme and then separately control API exposure.

Reply:
It's only used for the validation of the given GVs on the command line.

Reply:
> It's only used for the validation of the given GVs on the command line.

I think the interface to that method should be much tighter.  I'd define a local interface that does what you need and then if you want to use this as your impl, I guess I could live with it, but I'd really like to see the registry die as as thing.